### DISCUSSION TITLE
How to enable location access in project settings?

### DISCUSSION THREAD

**Discussion by Gelembjuk** [jevonmao/PermissionsSwiftUI#105] [Q&A] [answered]
When i tried to use the library PermissionsSwiftUI i got the error
"This app has attempted to access privacy-sensitive data without a usage description. The app's Info.plist must contain an “NSLocationWhenInUseUsageDescription” key with a string value explaining to the user how the app uses this data"

From web i can see that it is needed to add some info to the file Info.plist . You need to add keys
<key>NSLocationUsageDescription</key>
<string></string>

But in my project i do not have the file Info.plist. It seems this file was removed in Xcode 13 . 

So, what should i do to configure my project to allow access to location services?

**Comment by jevonmao** [comment]
@Gelembjuk Hi there. Checkout this answer on stack overflow: https://stackoverflow.com/questions/67896404/where-is-info-plist-in-xcode-13-missing-not-inside-project-navigator

**Comment by Gelembjuk** [comment, marked as answer]
Ok, i have found the solution.
It must be added in project -> targets -> Custom iOS Target properties 
You need to click "+" against one of options and below new empty row is added and you need to select Privacy - Location ...
